.github/workflows/shorebird_ci.yml

```diff
@@ -0,0 +1,41 @@
+name: shorebird_ci
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    name: 🐦 Test
+
+    env:
+      FLUTTER_STORAGE_BASE_URL: https://download.shorebird.dev
+
+    steps:
+      - name: 📚 Git Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Flutter
+        uses: subosito/flutter-action@v2
```

Comment:
Since we just need dart can we use the `setup-dart` action?

Reply:
Yup! Changed!